src/views/VirtualKeyboardView.vue

```diff
@@ -4,6 +4,7 @@ import VirtualPiano from '@/components/VirtualPiano.vue'
 import { useStateStore } from '@/stores/state'
 import { useScaleStore } from '@/stores/scale'
 import { computed } from 'vue'
+import type { NoteOnCallback } from 'xen-midi'
 
 defineProps<{
   noteOn: NoteOnCallback
```

Comment:
This prop now has the incorrect type (wrong number of arguments). You need to modify `App.vue` too.